ui/src/style/components/source-indicator.scss

```diff
@@ -1,7 +1,8 @@
 /*
-    Source Indicator component styles
-    ----------------------------------------------------------------
+    Source & Role Indicator component styles
+    ----------------------------------------------------------------------------
 */
+.role-indicator,
```

Comment:
The name of this file is no longer accurate

Reply:
👍 changed